### DISCUSSION TITLE
Name APs

### DISCUSSION THREAD

**Discussion by tvogel** [VREMSoftwareDevelopment/WiFiAnalyzer#381] [Ideas]
Hi!

I'd like to assign names to AP addresses so I can more easily see which AP is which. What do you think?

Regards,

Tilman

**Comment by computerconsultingpeople** [comment]
Love this app,  I came here to ask the exact same thing.  If it would be possible for us, for example, in settings to map the mac / bssid address of the WAP to a friendly name. Thanks for all your efforts thus far.

**Comment by VREMSoftwareDevelopment** [comment]
This is a feature request, not a discussion
